apps/ui/src/pages/applications/sections/applicant.tsx

```diff
@@ -18,12 +18,12 @@
  */
 
 import { zodResolver } from '@hookform/resolvers/zod';
+import { applicantInformationSchema, ApplicantInformationSchemaType } from '@pcgl-daco/validation';
```

Comment:
Can you add `type ApplicantInformationSchemaType` here? Jon and I were chatting yesterday, it's a helpful addition